content/references/rippled-api/public-rippled-methods/nft-methods/account_nfts.md

```diff
@@ -0,0 +1,118 @@
+---
+html: account_nfts.html
+parent: nft-methods.html
+blurb: Get a list of all NFTs for an account.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# account_nfts
+[[Source]](https://github.com/ripple/rippled/blob/xls20/src/ripple/rpc/handlers/AccountObjects.cpp "Source")
+{% include '_snippets/nfts-disclaimer.md' %}
+
+The `account_nfts` method returns a list of `NFToken` objects for the specified account.
+
+## Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+   "id": 1,
+   "command": "account_nfts",
+   "params": [{
+   		"account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH"
+   }]
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "account_nfts",
+    "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH"
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
```

Comment:
If there is a commandline format for these methods, please provide an example of that as well. If there is not a commandline method example, please add a note explaining that, such as the one on the [noripple_check page](https://xrpl.org/noripple_check.html).